### PR TITLE
landuse buffer

### PR DESCRIPTION
**Pull request by VignaPurohit**

### CODE REVIEW

File: source/docs/3/landuse_buffer.rst [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+    .. image:: /static/3/landuse_buffer/images/22.png
+       :align: center
+	   
+23.Select ``LandUse2020`` as :guilabel:`Input layer` and click :guilabel:`Run`.
+	
+    .. image:: /static/3/landuse_buffer/images/23.png
+       :align: center
+
+24. The spatial index is created for the layer. You can verify this by checking the property of the layer. Tight-click on the layer and click :guilabel:`Properties`.
```
Comment (spatialthoughts) [inline source/docs/3/landuse_buffer.rst:192]: Type. Change to Right-click

File: source/docs/3/landuse_buffer.rst [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+   .. image:: /static/3/landuse_buffer/images/20.png
+      :align: center
+
+21. A new layer called Calculated will be added to QGIS. Open the Attribute Table and verify that a new column called RESTRICTED with value Yes exists in this layer.
+
+   .. image:: /static/3/landuse_buffer/images/21.png
+      :align: center
+	  
+22. Next step is to do a Spatial Join to add this attribute to the original parcels layer based on which parcels intersect with the buffer zone. The ``LandUse2020`` layer has more than 100000 elements. We will add spatial indexing to the features to improve the performance and speedup the spatial join operation. 
+Check out the course `Spatial Indexing <https://courses.spatialthoughts.com/advanced-qgis.html#concept-spatial-indexing>`_ section to understand more about this.
```
Comment (spatialthoughts) [inline source/docs/3/landuse_buffer.rst:181]: "Check out the section on `Spatial Indexing `Spatial Indexing <https://courses.spatialthoughts.com/advanced-qgis.html#concept-spatial-indexing>`_ from our Advanced QGIS course.

File: source/docs/3/landuse_buffer.rst [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+   .. image:: /static/3/landuse_buffer/images/27.png
+      :align: center
+	  
+28. Select only the ``RESTRICTED`` field and click :guilabel:`OK`.
+
+   .. image:: /static/3/landuse_buffer/images/28.png
+      :align: center
+	  
+
+29. For Join Type, select ``Take attribute of the first matching feature only``. Next, click . . . next to Joined layer and select Save to File.
```
Comment (spatialthoughts) [inline source/docs/3/landuse_buffer.rst:218]: No space between dots. i.e. ...

**Review by spatialthoughts — COMMENTED** [review]
Few minor changes as suggested. Make the changes, commit them and the PR will auto update. Once done, reply to each comment as 'Done' and I will take a look.

File: source/docs/3/landuse_buffer.rst (hunk 1 above)
Comment (VignaPurohit) [reply source/docs/3/landuse_buffer.rst:192]: Done

File: source/docs/3/landuse_buffer.rst [hunk 4]
```diff
@@ -0,0 +1,235 @@
+Determining Landuse Buffer Zones(QGIS3)
+=====================================
+
+In this tutorial you will work with landuse data for a city and determine buffer zones around a particular land use. Such analysis is required to establish a corridor of restrictions around noise pollution or heavy traffic.
```
[the remark this answers is not in the document]
Comment (VignaPurohit) [reply source/docs/3/landuse_buffer.rst:4]: Done

File: source/docs/3/landuse_buffer.rst (hunk 2 above)
Comment (VignaPurohit) [reply source/docs/3/landuse_buffer.rst:181]: DOne

**Pull request merged**